docs/develop/make_it_your_own.md

```diff
@@ -65,37 +59,51 @@ If your logo isn't an svg, you still copy it to `static/images/`, but you need t
 + THEME_LOGO="images/my-logo.png"
 ```
 
-You also need to run `assets update` as above and additionally restart the server:
+Then, run `assets update` as above and additionally restart the server:
 
 ```bash
 ^C
 Stopping server and worker...
 Server and worker stopped...
 ```
 ```bash
-invenio-cli assets update
+invenio-cli assets update -d
 invenio-cli run
 ```
 
 !!! info "Re-run when invenio.cfg changes"
     All changes to `invenio.cfg` **MUST** be accompanied by a restart like the above to be picked up. This only restarts the server; it does not destroy any data.
 
-All other changes below follow the same pattern: add files and/or edit `invenio.cfg` and
-potentially execute `assets update` and/or rerun the server.
+This workflow stands for all `static/` files:
+
+- if you add a new file, run `invenio-cli assets update -d`
+- if you modify `invenio.cfg`, re-run `invenio-cli run` (because `invenio.cfg` has been symlinked above, you don't need to run `assets update`)
+- if you modify a previously symlinked file, you don't need to do anything
 
 
 ## Change the colors
 
 You might also be wondering: *How do I change the colors so I can make my instance look like my institution's theme?*
 
-We are going to change the top header section in the frontpage to apply our custom background color. Open the `assets/less/variables.less` file and edit it as below:
+We are going to change the top header section in the frontpage to apply our custom background color. It's a good example of the workflow for when `assets/` files change.
+
+Open the `assets/less/variables.less` file and edit it as below:
 
 ``` less
 @navbar_background_image: unset;
 @navbar_background_color: #000000; // Note this hex value is an example. Choose yours.
 ```
 
-Then, run the `invenio-cli assets update` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available styling variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/variables.less).
+Then, run the `invenio-cli assets update -d` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available styling variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/variables.less).
```

Comment:
comment: this also imports the variables from other modules e.g. invenio-rdm-records, but I guess is not needed to say that some variables might come from other modules.

Reply:
That's a good point. I've enhanced it a little.

(I had originally just added `-d` but GitHub highlights the whole line :sweat_smile: )